<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progetto di Francesco</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pagina_principale">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>401</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="go_pag_autori">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(24,119,242);</string>
         </property>
         <property name="text">
          <string>Aggiungi autori</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="go_pag_riviste">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(24,119,242);</string>
         </property>
         <property name="text">
          <string>Aggiungi riviste</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="go_pag_conferenze">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(24,119,242);</string>
         </property>
         <property name="text">
          <string>Aggiungi conferenze</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pagina_autori">
     <widget class="QListWidget" name="lista_autori">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>221</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="aggiungi_autore">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,119,242);</string>
      </property>
      <property name="text">
       <string>Aggiungi</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>221</width>
        <height>74</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_cognome">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Cognome</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_nome">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="linea_nome">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="linea_cognome">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_id_autore">
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="id_autore">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pulsante_torna_menu">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,119,242);</string>
      </property>
      <property name="text">
       <string>Menu principale</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagina_riviste">
     <widget class="QCalendarWidget" name="calendario_rivista">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>331</width>
        <height>183</height>
       </rect>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::NoHorizontalHeader</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="lista_riviste">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>531</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pulsante_aggiungi_rivista">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,119,242);</string>
      </property>
      <property name="text">
       <string>Aggiungi Rivista</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pulsante_tornaalmenu">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,119,242);</string>
      </property>
      <property name="text">
       <string>Menu principale</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>91</y>
        <width>189</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="nome_rivista">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="acronimo_rivista">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Acronimo</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="editore_rivista">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Editore</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="volume_rivista">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLineEdit" name="inserisci_nome"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="inserisci_acronimo"/>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="inserisci_editore"/>
         </item>
         <item row="3" column="0">
          <widget class="QSpinBox" name="inserisci_volume">
           <property name="maximum">
            <number>999999999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pagina_conferenze">
     <widget class="QCalendarWidget" name="calendario_conferenze">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>312</width>
        <height>183</height>
       </rect>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::NoHorizontalHeader</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::NoVerticalHeader</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="vai_al_menu_principale">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,119,242);</string>
      </property>
      <property name="text">
       <string>Menu principale</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pulsante_aggiungi_conferenza">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>191</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,119,242);</string>
      </property>
      <property name="text">
       <string>Aggiungi conferenza</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lista_conferenze">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>521</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>191</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_nome_conf">
         <property name="text">
          <string>Nome</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="linea_nome_conferenza"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_acronimo_conf">
         <property name="text">
          <string>Acronimo</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="linea_acronimo_conferenza"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_luogo_conf">
         <property name="text">
          <string>Luogo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="linea_luogo_conferenza"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_part_conf">
         <property name="text">
          <string>Partecipanti</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="num_partecipanti_conferenza"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
